client/components/animations.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Canvas from './canvas';
+
+export default function Animations(props) {
+  return (
+    <>
+      <Canvas />
+    </>
+  );
+}
```

Comment:
We can keep this module for now since you'll likely want it for testing, but we should remove it from pages/components that don't normally have it so we aren't yucking up the master.